docs/docs/spark-ddl.md

```diff
@@ -425,6 +427,9 @@ To order within each task, not across tasks, use `LOCALLY ORDERED BY`:
 ALTER TABLE prod.db.sample WRITE LOCALLY ORDERED BY category, id
 ```
 
+!!! note
+    Write distribution of `hash` will be changed to `none` by this. Please use `ALTER TABLE .. WRITE DISTRIBUTED BY PARTITION LOCALLY ORDERED BY` to preserve the write distribution.
```

Comment:
How about just ```This will request a write distribution of `none`.```

Not sure if the other parts are necessary, if they are mentioned below?

Reply:
I think it's necessary since people might not make connections between the two.